src/test/java/seedu/duke/template/LrTemplateTest.java

```diff
@@ -0,0 +1,74 @@
+package seedu.duke.template;
+
+import org.junit.jupiter.api.Test;
+import seedu.duke.DukeException;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class LrTemplateTest {
+    public static final double DELTA = -5 * 10 ^ -5;
```

Comment:
Probably don't want to XOR here right?
```suggestion
    public static final double DELTA = 1e-15;
```

Reply:
Oops, my bad! I'll use the pow function instead